StaticAnalyzer/views/ios/code_analysis.py

```diff
@@ -70,7 +76,6 @@ def ios_source_analysis(src):
             'urlnfile': url_n_file,
             'domains': domains,
             'emailnfile': email_n_file,
-            'source_type': source_type,
```

Comment:
We need a logic to detect the kind of source here.
1. Swift
2. Objective-C
3. Swift, Objective-C

Reply:
Done